tests/utils/mock.rs

```diff
@@ -6,14 +6,14 @@ use std::{
     fs::create_dir,
 };
 
-use crate::utils::{fixtures, VerifierWallet};
+use crate::utils::{VerifierWallet, fixtures};
```

Comment:
Question -- do changes like this where the parameters are resorted mean that the wrappers have to change their calls?  Or are such changes only on imports.  This example seems to be an import, but I thought I saw others that  that were calls with ordering changes.